### ISSUE TITLE
Missing information in manual for working setup iocage, vnet, bridge, pf, FreeBSD 12 ?

### ISSUE THREAD

**Issue by codegrau** [iocage/iocage#1051]
Hi there, 
I read the faq / manual and tried to configure and setup a vnet jail with iocage 1.2 with a freshly installed FreeBSD 12.0-RELEASE but no luck (no internet in jail). So I wonder: it'd be great to have a how to for a minimal working setup with vnet enabled jails. I have problems understanding where (and how) to NAT.

Classic setup: 
em0 = external interface with DHCP, gets IP from hoster (say it's 66.77.88.99)
lo0 = loopback

I followed this manual page (https://iocage.readthedocs.io/en/latest/networking.html) - tried the vnet setup with 12.0-RELEASE (so everything should be in the kernel, right?). 

1. bridge0 is up but doesn't get an IP (shouldn't that be configured and set by me in rc.conf on host like `ifconfig_bridge0="inet 10.10.10.1 netmask 255.255.255.0 addm em0 up"`?)
2. What is "jail's default gateway"? shouldn't that be the IP of bridge0? If not: where do i get or set this on the host or where do I find this defaultrouter when external IP is handed out by third party?
3. "Tips and Tricks: Routes - Be sure the default gateway knows the route back to the VNET subnets." I really don't know how to accomplish this?
4. Don't I need a NAT in PF for this to work? What would be the minimal pf-rules setup to get internet access inside the jail?

Long story short: 
No luck with getting internet access inside jail! 
I'd really appreciate a working example with vnet (outside em0 IP range - say 10.1.1.0/24), bridge configuration and minimal pf rules (nat?) and anything that's missing for a minimal setup for beginners. 

Would be thankful if you could post the regarding lines of rc.conf / pf.conf / iocage command / resolv.conf inside and outside of a vnet jail.

Looking forward to hear from you.
